src/main/java/org/opentripplanner/transit/raptor/rangeraptor/debug/DebugHandlerPatternRideAdapter.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner.transit.raptor.rangeraptor.debug;
+
+import org.opentripplanner.transit.raptor.api.request.DebugRequest;
+import org.opentripplanner.transit.raptor.api.view.ArrivalView;
+import org.opentripplanner.transit.raptor.rangeraptor.WorkerLifeCycle;
+import org.opentripplanner.transit.raptor.rangeraptor.multicriteria.PatternRide;
+
+import java.util.LinkedList;
+
+/**
+ * PatternRide debug adapter.
```

Comment:
This comment repeats the name of the class. It should be removed or expanded. In discussion we thought a short description of this debugging system should be placed in the raptor design documents to help future readers understand what all this code is for.

Reply:
I have added a section on debugging in the Raptor `package.md`. I also added JavaDoc to som of the main/important classes and removed JavaDoc witch was not adding any value.